#!/usr/bin/python3
def raise_exception_msg(message=""):
    """raise name exception with message"""
    raise NameError(message)
